src/pages/AdminDashboard/CreateRoomForm/createRoomTools.ts

```diff
@@ -0,0 +1,71 @@
+import { parsePhoneNumber } from 'libphonenumber-js'
+import { UNITS } from '../Reminders/NotificationSelector'
+
+export const mapDestinationsToInputField = (destinations) => {
+    const values = [] as any
+    for (const destination of destinations) {
+        const [value] = Object.values(destination)
+        values.push(value)
+    }
+
+    return values
+}
+
+export const getRemindAt = (startAt, notification) => {
+    let remindBeforeTimestampSecs = 0
+    switch (notification.unit) {
+        case UNITS.mins:
+            remindBeforeTimestampSecs = notification.number * 60
+            break
+        case UNITS.hours:
+            remindBeforeTimestampSecs = notification.number * 60 * 60
+            break
+        case UNITS.days:
+            remindBeforeTimestampSecs = notification.number * 60 * 60 * 24
+            break
+    }
+
+    return startAt - remindBeforeTimestampSecs
+}
+
+export const validateEmail = (email) => {
+    const regexp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+    return regexp.test(email)
+}
+
+export const validatePhoneNumber = (phoneNumber) => {
+    let isValid = true
+
+    try {
+        parsePhoneNumber(phoneNumber, 'FR')
+    } catch (error) {
+        isValid = false
+    }
+
+    return isValid
+}
+
+export const isParticipantValid = (participant) => {
+    return validateEmail(participant) || validatePhoneNumber(participant)
+}
+
+export const isNumeric = (destination) => {
+    for (let char of destination) {
+        if (char >= '0' && char <= '9') continue
```

Comment:
what if the first char is a "+"?

Reply:
validation is currently commented in order to be replaced with the use of `yup`

Reply:
i had already ask here and gave a suggestion https://github.com/Instant-Visio/InstantVisio-WebApp/pull/455#discussion_r550265953